scripts/partition_manager/partition_manager.rst

```diff
@@ -717,11 +717,14 @@ The sizes of each partition are determined by the associated :file:`pm.yml` file
 Static configuration
 ********************
 
+The Partition Manager provides static configuration that lets you define static partitions.
+This feature can be used with single-image applications, but its most common use case is with multi-image ones.
+
 By default, the Partition Manager dynamically places the partitions in memory.
-However, if you have a deployed product that consists of multiple images, where only a subset of the included images can be upgraded through a firmware update mechanism, the upgradable images must be statically configured.
+However, there are cases where a deployed product can consist of multiple images, and only a subset of these images can be upgraded through a firmware update mechanism.
+In such cases, you must statically configure the upgradable images using the Partition Manager's static configuration.
```

Comment:
`you must rely on static configuration of partitions to ensure that all of the built images are compatible.`